templates/password-enhancements-policy-constraints.html.twig

```diff
@@ -0,0 +1,24 @@
+{#
+/**
+ * @file
+ * Template for the frontend validation.
+ *
+ * Please note that if you need to change the structure, an element with the ID
+ * of 'password-policy-constraints' has to exist and it must contain the
+ * elements with .constraint without changing their class, data-* and ID
+ * attributes. Of course it is possible to add more classes as needed.
+ *
+ * Available variables:
+ *   - constraints (Drupal\password_enhancements\Entity\Constraint[]): Constraint entities
+ *       keyed by their IDs.
+ *   - descriptions (string[]): Initial description for each constraint keyed by
+ *       the entity's ID.
+ */
+#}
+
+<fieldset id="password-policy-constraints" {{ hidden ? 'style="display: none;"' : '' }}>
```

Comment:
Not sure if fieldset is the best choice for a render element. Ask feta about this.
https://www.drupal.org/node/1852020

Reply:
Yes, Details element is preferred.

Reply:
The fieldset is not collapsible by default, details are, and it is not possible to say that it shouldn't be collapsible. I would rather not make this field collapsible, unless it is something that helps the user on the UI :) but I doubt that.

Also the default theme for the fieldset is looks better than details which just looks like that it hangs in the air.